src/components/ContextBar.js

```diff
@@ -59,7 +59,8 @@ export const ContextBar = ({
     [tools.Jupyter.label, () => img({ src: jupyterLogo, style: { height: 45, width: 45 }, alt: '' })],
     [tools.Galaxy.label, () => img({ src: galaxyLogo, style: { height: 14, width: 45 }, alt: '' })],
     [tools.Cromwell.label, () => img({ src: cromwellImg, style: { width: 45 }, alt: '' })],
-    [tools.RStudio.label, () => img({ src: rstudioSquareLogo, style: { height: 45, width: 45 }, alt: '' })]
+    [tools.RStudio.label, () => img({ src: rstudioSquareLogo, style: { height: 45, width: 45 }, alt: '' })],
+    [tools.Azure.label, () => img({ src: jupyterLogo, style: { height: 45, width: 45 }, alt: '' })]
```

Comment:
I find `Azure` as the tool name to be confusing. I would recommend changing both the front-end and back-end code to call this Juptyer_on_Azure.

I do wonder though… in the long term, is the vision that Leo will be able to create Juptyer on either GCP or Azure, regardless of which cloud platform Leo itself is running on? If that's not the vision and there will always be the constraint that Leo creates the tool on the cloud platform that Leo itself is running on, then why not make the APIs so that you tell Leo to create Jupyter, and it does it on the correct cloud platform? I suppose the issue with that would be if different options need to be specified by the user for creating Jupyter on Azure vs. GCP… if so, the front end code does need to be aware at some point so it knows which options to present, though I would still think the server API endpoint could be the same, just with different parameters passed.

Reply:
You could make tools.Azure.label "Jupyter" though to help reduce the confusion. It is odd to see "Launch Azure".

![image](https://user-images.githubusercontent.com/484484/166685829-7e5f44c3-d27c-479f-908a-b99a774776e5.png)